ignition-math4.rb

```diff
@@ -19,7 +19,10 @@ class IgnitionMath4 < Formula
   depends_on "ignition-cmake0"
 
   conflicts_with "ignition-math2", :because => "Symbols collision between the two libraries"
-  conflicts_with "ignition-math3", :because => "Symbols collision between the two libraries"
+  # TODO(clalancette): Temporarily disabling this conflicts_with so that CI passes for RNDF;
+  # once we land the changes to switch ignition-rndf over to ignition-cmake, this can be
+  # re-enabled.
+  #conflicts_with "ignition-math3", :because => "Symbols collision between the two libraries"
```

Comment:
I think this might not be a conflict actually since we implemented inline namespaces in ignition-math4

I think we can just delete the line

Reply:
Removed in 4f2643e.